README.md

```diff
@@ -257,6 +259,16 @@ Notes:
 
 ### Special classes within teacher content
 
+####Teacher Content with NGSS tag
+
+```html
+
+<note class="os-teacher">
+  <label>Teacher Edition</label>
+  <p>NGSS HS-PS2-1:<span class="ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze
```

Comment:
I tend towards leaving it as-is, based on the possibility of inconsistent handling by the XML team who has to locate the second colon in a paragraph. However, if Phil feels strongly, we could definitely change it in the TL and see how the XML team handles it in the first chapter they send back?